vent/io/devices/valves.py

```diff
@@ -209,16 +216,16 @@ def _load_valve_response(self, response_path):
         if response_path is not None:
             response_array = np.load(response_path)
         else:
-            response_array = np.linspace([0, 0, 0], [1, 1, 1], num=101)
+            response_array = np.linspace([0, 0, 0], [100, 1, 1], num=101)
         self._response_array = response_array
 
 
 class SimOnOffValve(SolenoidBase):
     """ stub: a simulated on/off valve"""
 
-    def __init__(self, form='Normally Closed', pig=None):
+    def __init__(self, pin=None, form='Normally Closed', pig=None):
         super().__init__(form=form)
-        self.state = 0
+        self.state = 0 if form == 'Normally Closed' else 1

```

Comment:
should explicitly check other condition (ie. if form == "Normally Open") and except if neither.

Reply:
Parent class does that (line 27) in @form.setter